### PR TITLE
[IOTDB-1645] Feature/optimize query & verficiation

### PR DESCRIPTION
**Pull request by stormbroken**
In this PR, I optimize the logic of query generate and support point verification between two database. The details are as follows.
1. I modify the timestamp generate logic of Q1-Q10: timestamp increment for each operation separately.
2. I support point verification between two database: compare between each other, use IS_POINT_COMPARISON to use this function.
3. Add check of config: IS_COMPARISON and IS_POINT_COMPARISON only used in double write mode and can't be both true

### CODE REVIEW

**Comment by stormbroken** [comment]
And fix not aligned sensor for timescaleDB

File: core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/client/generate/SyntheticClient.java [hunk 1]
```diff
@@ -57,115 +66,209 @@ public SyntheticClient(int id, CountDownLatch countDownLatch, CyclicBarrier barr
...
+                  }
+                  // 顺序比较
+                  for (int i = 0; i < point1; i++) {
+                    List<Object> record1 = records1.get(i);
+                    List<Object> record2 = records2.get(i);
+                    for (int j = 0; j < record1.size(); j++) {
+                      String r1 = String.valueOf(record1.get(j));
+                      String r2 = String.valueOf(record2.get(j));
+                      if (!r1.equals(r2)) {
+                        isError = true;
```
Comment (richard-yuan16) [inline core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/client/generate/SyntheticClient.java:155]: As soon as isError=true, we can break the loop, unless you need to count how many different points.

Comment (stormbroken) [reply core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/client/generate/SyntheticClient.java:155]: OK, I think fast-fail is a good idea, I will fix it.

**Comment by stormbroken** [comment]
I change the compare in fail-fast way

**Pull request merged**